src/algebra/category/Group/equivalence_Group_AddGroup.lean

```diff
@@ -0,0 +1,101 @@
+/-
+Copyright (c) 2022 Jujian Zhang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jujian Zhang
+-/
+import algebra.category.Group.basic
+
+/-!
+# Equivalence between `Group` and `AddGroup`
+
+This file contains two equivalence:
+* `Group_AddGroup_equivalence` : the equivalence between `Group` and `AddGroup` by sending
+  `X : Group` to `additive X` and `Y : AddGroup` to `multiplicative Y`.
+* `CommGroup_AddCommGroup_equivlance` : the equivalence between `CommGroup` and `AddCommGroup` by
+  sending `X : CommGroup` to `additive X` and `Y : AddCommGroup` to `multiplicative Y`.
+-/
+
+open category_theory
+
+namespace Group
+
+/--
+The functor `Group ⥤ AddGroup` by sending `X ↦ additive X` and `f ↦ f`.
+-/
+@[simps] def to_AddGroup : Group ⥤ AddGroup :=
+{ obj := λ X, AddGroup.of (additive X),
+  map := λ X Y, monoid_hom.to_additive }
+
+end Group
+
+namespace CommGroup
+
+/--
+The functor `CommGroup ⥤ AddCommGroup` by sending `X ↦ additive X` and `f ↦ f`.
+-/
+@[simps] def to_AddCommGroup : CommGroup ⥤ AddCommGroup :=
```

Comment:
Please apply the same changes you applied to `to_AddGroup` also to `to_AddCommGroup`, `to_Group` and `to_CommGroup`.

Reply:
done